api/query_test.go

```diff
@@ -411,3 +411,41 @@ func TestQueryErrorsTimeout(t *testing.T) {
 		t.Error(diffs)
 	}
 }
+
+func TestResponseCompression(t *testing.T) {
+	// Stand up the server
+	store := mock.EntityStore{
+		ReadEntitiesFunc: func(entityType string, q query.Query, f etre.QueryFilter) ([]etre.Entity, error) {
+			return testEntitiesWithObjectIDs, nil
+		},
+	}
+	server := setup(t, defaultConfig, store)
+	defer server.ts.Close()
+
+	// Make the http request.
+	etreurl := server.url + etre.API_ROOT + "/entities/" + entityType +
+		"?query=" + url.QueryEscape("foo=bar")
+	req, err := http.NewRequest("GET", etreurl, nil)
+	if err != nil {
```

Comment:
Nit: use the testify `assert` and `require` packages to improve readability.

Reply:
I'm not doing this right now because none of the etre tests use these and I'm favoring consistency. I think at some point we should go through and update all the tests to use assert and require, but I didn't want to introduce one or two one-offs.